Bindings/Java/Matlab/Utilities/opensimC3D.m

```diff
@@ -0,0 +1,132 @@
+            import org.opensim.modeling.*
+            % Use a c3dAdapter to turn read a c3d file
+            tables = C3DFileAdapter().read(path2c3d);
+            % get the marker and force data into OpenSim tables
+            obj.markers = tables.get('markers');
+            obj.forces = tables.get('forces');
+        end            
+        function rate = getRate_marker(obj)
+            % Get the capture rate used for the Marker Data
+            rate = str2double(obj.markers.getTableMetaDataAsString('DataRate'));
+        end
+        function rate = getRate_force(obj)
+            % Get the capture rate used for the Force Data
+            rate = str2double(obj.forces.getTableMetaDataAsString('DataRate'));
+        end
+        function n = getNumTrajectories(obj)
+            % Get the number of markers in the c3d file
+            n = obj.markers.getNumColumns;
+        end
+        function n = getNumForces(obj)
+            % Get the number of forceplates in the c3d file
+            n = (obj.forces.getNumColumns)/3;
+        end
+        function t = getStartTime(obj)
+            % Get the start time of the c3d file
+            t = obj.markers.getIndependentColumn().get(0);
+        end
+        function t = getEndTime(obj)
+            % Get the end time of the c3d file
+            t = obj.markers.getIndependentColumn().get(obj.markers.getNumRows - 1);
+        end
+        function name = getFileName(obj)
+            % Get the name of the c3d file
+            [filedirectory, name, extension] = fileparts(obj.path);
+        end
+        function filedirectory = getDirectory(obj)
+            % Get the directory path for the c3d file
+            [filedirectory, name, extension] = fileparts(obj.path);
+        end
+        function table = getTable_markers(obj)
+            table = obj.markers();
+        end
+        function table = getTable_forces(obj)
+            table = obj.forces();
+        end
+        function [markerStruct, forcesStruct] = getAsStruct(obj)
+            % Convert the OpenSim tables into Matlab Structures
+            markerStruct = osimTableToStruct(obj.markers);
+            forcesStruct = osimTableToStruct(obj.forces);
+            disp('Maker and force data returned as Matlab Structs')
+        end
+        function rotateData(obj,axis,value)
+            % Method for rotating marker and force data stored in OpenSim 
+            % tables
+            if ~ischar(axis) 
+               error('Axis must be either x,y or z')
+            end
+            if ~isnumeric(value)
+                error('value must be numeric (90, -90, 270, ...')
+            end
+            
+            % rotate the tables
+            markers_rot = obj.rotateTable(obj.markers, axis, value);
+            forces_rot = obj.rotateTable(obj.forces, axis, value);
+        
+            % set the internal tables to the new rotated ones
+            obj.setMarkers(markers_rot)
+            obj.setForces(forces_rot)
```

Comment:
It seems likely to me that these calls are unnecessary (that `obj.rotateTable()` rotates the tables in place).